### DISCUSSION TITLE
Make Poetry compatible with default [project] attributes in pyproject.toml

### DISCUSSION THREAD

**Discussion by evan0greenup** [python-poetry/poetry#7414] [Ideas]
The project attribute of poetry project is under `[tool.poetry]`.

However, the majority of other python project management tools is all put project information under `[project]`.

It would be nice to make poetry compatible with default python project information in `pyproject.toml`

**Comment by Secrus** [comment]
This is being worked on. You can see https://github.com/python-poetry/roadmap/issues/3 for details

**Comment by neersighted** [comment]
[poetry#3332](https://github.com/python-poetry/poetry/issues/3332)
